runtime/onert/backend/train/ops/PoolLayer.cc

```diff
@@ -52,8 +52,8 @@ class MaxPool2D final : public TrainingKernelRegistry
   const IPortableTensor *_output;
   nnfw::cker::PoolParams _op_params;
```

Comment:
```suggestion
  nnfw::cker::PoolParams _op_params{};
```

If several suggestions requires much changes, and you're busy now, this may be a simple solution. It will initialize `PoolParams` with zero.